include/tvm/runtime/vm/vm.h

```diff
@@ -174,7 +174,7 @@ class VirtualMachine : public runtime::ModuleNode {
    * \brief load the executable for the virtual machine.
    * \param exec The executable.
    */
-  virtual void LoadExecutable(Executable* exec);
+  virtual void LoadExecutable(const ObjectRef& exec);
```

Comment:
consider change to `ObjectPtr<Executable>` which can be constructed via https://github.com/apache/tvm/blob/main/include/tvm/runtime/object.h#L877

Reply:
It was advice from @jroesch to use ObjectRef. And it seems the simplest way from possible ones. I agree that using of ObjectPtr<Executable> is preferable but to get it correctly is not so simple task (see below).